staging/src/github.com/kubeedge/mapper-framework/_template/mapper/data/dbmethod/mysql/client.go

```diff
@@ -0,0 +1,93 @@
+package mysql
```

Comment:
Add KubeEdge License header for all new files.

Reply:
Ihave added it.